strawberryfields/backends/gaussianbackend/gaussiancircuit.py

```diff
@@ -357,15 +357,15 @@ def loss(self, T, k):
         self.mmat[:, k] = self.mmat[k]
         self.mean[k] = sqrtT*self.mean[k]
 
-    def thermal_loss(self, T, nth, k):
+    def thermal_loss(self, T, nbar, k):
         r""" Implements the thermal loss channel in mode k by amplitude loss amount \sqrt{T}
         unlike the loss channel, here the ancilliary mode that goes into the second arm of the
         beam splitter is prepared in a thermal state with mean photon number nth """
```

Comment:
I cleaned up this docstring in `master`. Let's remember to double check that the merge doesn't revert back to the old one